bin/all_sky_search/pycbc_plot_kde_vals

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+import h5py, argparse
+import matplotlib.pyplot as plt
+from matplotlib.colors import LogNorm
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument('--signal-file')
+parser.add_argument('--template-file', required=True)
+parser.add_argument('--param', nargs='+', required=True, help='mchirp, eta, chi_eff')
+parser.add_argument('--plot-type', choices=['kde_vs_param', 'param_vs_param'])
+parser.add_argument('--which-kde', choices=['signal_kde','template_kde', 'ratio_kde'])
+parser.add_argument('--plot-dir', required=True)
+parser.add_argument('--verbose', action='count')
+args = parser.parse_args()
+
+signal_dataset = h5py.File(args.signal_file, 'r')
+template_dataset = h5py.File(args.template_file, 'r')
+param0 = template_dataset[args.param[0]][:]    #mchirp,eta,chi_eff
+param1 = template_dataset[args.param[1]][:]
+param2 = template_dataset[args.param[2]][:]
```

Comment:
I don't think the code uses param2 at all!  Also, if the user wants to plot kde vs param it should not be necessary to make 2 inputs to the param option.